core/trino-main/src/main/java/io/trino/execution/scheduler/TopologyAwareNodeSelector.java

```diff
@@ -23,6 +23,7 @@
 import io.trino.execution.RemoteTask;
 import io.trino.metadata.InternalNode;
 import io.trino.metadata.InternalNodeManager;
+import io.trino.metadata.NodeMap;
```

Comment:
unused?

Reply:
NodeMap was moved to a new package, `io.trino.execution.scheduler -> io.trino.metadata` hence the imports